### ISSUE TITLE
db.setex ( TypeError )

### ISSUE THREAD

**Issue by TeaByte** [AYMENJD/Kvsqlite#1]
```py
db.setex("test", "This key has a lifetime of 60 seconds", 80)
  File "c:\Users\Yazan\Desktop\deno-tast\test\Kvsqlite\kvsqlite\sync\client.py", line 103, in setex        
           ^^^^^^^^
TypeError: '>=' not supported between instances of 'str' and 'int'
```

at readme.md it says:
```py
await db.setex(key, "This key has a lifetime of 60 seconds", 60)
# key, value, ttl
```

while lib take args in deferent order:
```py
async def setex(self, key: str, ttl: int, value) -> bool:
# key, ttl, value
```

It's just the wrong order!

**Comment by AYMENJD** [comment]
That's a README bug. The library behavior is intended.

**Comment by TeaByte** [comment]
oh.
I thought the other way 
Bcz its weird to be (key, ttl, value) 🤷

**Comment by AYMENJD** [comment]
Redis already does that. Which i made it for easy switching.

**Comment by TeaByte** [comment]
Well. I have fixed the pull request to just edit the readme.md

**Comment by TeaByte** [comment]
Sorry about the conflict
My editor format the markdown

**Comment by AYMENJD** [comment]
This was fixed in #2.
